opendbc/car/hyundai/interface.py

```diff
@@ -155,9 +155,12 @@ def _get_params_sp(stock_cp: structs.CarParams, ret: structs.CarParamsSP, candid
         ret.flags |= HyundaiFlagsSP.ENHANCED_SCC.value
 
     if ret.flags & HyundaiFlagsSP.ENHANCED_SCC:
-      stock_cp.safetyConfigs[-1].safetyParam |= HyundaiSafetyFlags.FLAG_HYUNDAI_ESCC.value
+      ret.safetyParam |= HyundaiSafetyFlagsSP.ESCC
```

Comment:
```suggestion
      ret.safetyParam |= HyundaiSafetyFlagsSP.ESCC.value
```

Don't need `.value` ?

Reply:
For `HyundaiSafetyFlagsSP`, since we defined them directly as integer constants (not as an Enum), we don't need the `.value`. The constant `ESCC ` is already the integer value that can be used directly in bitwise.